ktor-http/common/src/io/ktor/http/ContentTypes.kt

```diff
@@ -121,6 +121,7 @@ class ContentType private constructor(val contentType: String, val contentSubtyp
         val Any = ContentType("application", "*")
         val Atom = ContentType("application", "atom+xml")
         val Json = ContentType("application", "json")
+        val JsonHal = ContentType("application", "json+hal")
```

Comment:
According to spec, the type is `application/hal+json`, see e.g. http://stateless.co/hal_specification.html.

Reply:
You are right, thank you, I haven't checked it. Anyway, I fixed it.